collector/collector_test.go

```diff
@@ -358,7 +519,7 @@ install:
 				Expect(err).ToNot(HaveOccurred())
 			})
 
-			It("should be the same as just one of them", func() {
+			It("remain duplicated, and are the responsibility of the user", func() {
```

Comment:
:warning: This will be the main change from before, since now we will have duplicates and the user needs to be in charge of them